AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Campaigns/CampaignSummaryQueryHandlerTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿using AllReady.Areas.Admin.Features.Campaigns;
+using AllReady.Models;
+using Shouldly;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace AllReady.UnitTest.Areas.Admin.Features.Campaigns
+{
+    public class CampaignSummaryQueryHandlerShould : InMemoryContextTest
```

Comment:
you've changed the name of the class to be correct, but the name of the .cs file in disk still says `CampaignSummaryQueryHandlerTests.cs`. 
![image](https://cloud.githubusercontent.com/assets/2428993/18725246/7beffa38-800d-11e6-940d-7629bb314b6e.png)
please update it to the correct name.